pyproject.toml

```diff
@@ -0,0 +1,131 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=64.0.0",
+    "tomli  ; python_version < '3.11'",
+    "versioneer>=0.24",
+]
+
+[project]
+name = "dask-cuda"
+dynamic = [
+    "version",
+]
+description = "Utilities for Dask and CUDA interactions"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license= { text = "Apache-2.0" }
+requires-python = ">=3.8"
+dependencies = [
+    "dask >=2022.9.2",
+    "distributed >=2022.9.2",
+    "pynvml >=11.0.0",
+    "numpy >=1.16.0",
+    "numba >=0.54",
+]
+classifiers=[
+    "Intended Audience :: Developers",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+
+[project.scripts]
+dask-cuda-worker = "dask_cuda.cli.dask_cuda_worker:go"
+dask-config = "dask_cuda.cli.dask_config:go"
+
+[project.optional-dependencies]
+docs = [
+    "numpydoc",
+    "sphinx",
+    "sphinx-click",
+    "sphinx-rtd-theme",
```

Comment:
Some of those had version requirements (see https://github.com/rapidsai/dask-cuda/pull/1035/files#diff-2c9c11d26b09b8afde329980309d967121543a456e4592c76886a20b5cf56c90), are those not required anymore?

Reply:
Those seem to be pinned very tightly to specific versions (that are likely pretty outdated at this point). So no don't think we want those here. We could set lower bounds if we wanted them

Reply:
Added those as lower bounds

The challenge with having them pinned that tightly generally is they will fall out of date, have unfixed bugs, and be incompatible with newer Python & Sphinx versions (the latter also not being pinned currently)